template.tpl

```diff
@@ -200,34 +268,18 @@ ___WEB_PERMISSIONS___
   {
     "instance": {
       "key": {
-        "publicId": "get_cookies",
+        "publicId": "inject_script",
         "versionId": "1"
       },
       "param": [
         {
-          "key": "cookieNames",
+          "key": "urls",
           "value": {
             "type": 2,
             "listItem": [
               {
                 "type": 1,
-                "string": "_bamls_usid"
-              },
-              {
-                "type": 1,
-                "string": "_bamls_cuid"
-              },
-              {
-                "type": 1,
-                "string": "_bamls_shid"
-              },
-              {
-                "type": 1,
-                "string": "_bamls_seid"
-              },
-              {
-                "type": 1,
-                "string": "_bamls_caid"
```

Comment:
Don't we need to declare these cookies anymore?

Reply:
I don't think we've ever needed the 'get_cookies' permission.
We read cookies through the tracking library methods, not through the tag.

This permission is only needed when we read cookies using [getCookieValues](https://developers.google.com/tag-manager/templates/api#getcookievalues) method inside the GTM tag sandbox.